### ISSUE TITLE
How can I change Scenario's Name color?

### ISSUE THREAD

**Issue by toshiossada** [eBay/flutter_glove_box#128]
How can I change Scenario's Name color?

![image](https://user-images.githubusercontent.com/2637049/141370904-db907a5a-7a87-4570-80fa-803a448a6eb9.png)

**Comment by coreysprague** [comment]
I don't believe that option is available right now. If we wanted to add support for this, we could allow for an optional builder func to be supplied to the GoldenBuilder @toshiossada

**Comment by toshiossada** [comment]
perhaps the error it's because font family, cause I changed color and still blurry
![image](https://user-images.githubusercontent.com/2637049/142302329-c4a59ae8-ab15-4391-99d6-7bf977fac934.png)

**Comment by jamesblasco** [comment]
@coreysprague That would be great to have! Also to limit the text to a 1 line max
